### ISSUE TITLE
Add component accessibility status to website

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon-website#325]
It would be great with the work that @dakahn is doing in https://github.com/IBM/carbon-components/issues/1308 could be represented on our website. We often get asked the question "is Carbon accessible" and it would be nice to have a resource that talks about:

- If Carbon is accessible and what that means
- What our process is for determining accessibility, including:
  - How we test
  - What did we test
  - When did we last test
  - Why did we test this way
